<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ECATMainWindow</class>
 <widget class="QMainWindow" name="ECATMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1377</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COWA-LUGE-CALIB</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/calibration.png</normaloff>:/images/images/calibration.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="connectAct"/>
   <addaction name="autocalibAct"/>
   <addaction name="laserdetectAct"/>
   <addaction name="setupAct"/>
   <addaction name="calibrationAct"/>
  </widget>
  <action name="openAct">
   <property name="icon">
    <iconset>
     <normalon>:/images/images/insert.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="newAct">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="saveAct">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="printAct">
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="undoAct">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAct">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="connectAct">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/connect.png</normaloff>:/images/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="setupAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/setup.png</normaloff>:/images/images/setup.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Setup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="laserdetectAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/dashboard.png</normaloff>:/images/images/dashboard.png</iconset>
   </property>
   <property name="text">
    <string>&amp;LaserDetect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionHello">
   <property name="text">
    <string>Hello</string>
   </property>
  </action>
  <action name="actionWorld">
   <property name="text">
    <string>World</string>
   </property>
  </action>
  <action name="calibrationAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/calibration.png</normaloff>:/images/images/calibration.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Calibration</string>
   </property>
   <property name="toolTip">
    <string>Calibration</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="autocalibAct">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/notvisible.png</normaloff>:/images/images/notvisible.png</iconset>
   </property>
   <property name="text">
    <string>&amp;autocalib</string>
   </property>
   <property name="toolTip">
    <string>AutoCalib</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
